---
title: Research
layout: default
permalink: /research/
---

<head><script src="//code.simplesvg.com/1/1.0.0-beta5/simple-svg.min.js"></script></head>

<div class="page-heading">
  <h2 class="page-title">Research</h2>
</div>

<div class="post-box">
	<div class="icon-on-left">
		<simple-svg class="simple-svg" data-icon="uil-cloud-database-tree" style="font-size: 5em; margin-right: .3em; float: left;"></simple-svg>
		<b>Phylogenetic classification of long read sequences under the TreeSAPP pipeline</b>
	</div>
	<div>
		<p>View my thesis <a href="https://open.library.ubc.ca/cIRcle/collections/undergraduateresearch/52966/items/1.0378444">here</a>!</p>
		<p>TreeSAPP (Tree-based Sensitive and Accurate Protein Profiler) is an analysis pipeline designed to functionally and taxonomically classify protein and nucleotide sequences using marker genes and phylogenetic methods. Currently, TreeSAPP supports short read sequencing data (e.g. Illumina), but does not support long reads from newer sequencing platforms (e.g. Nanopore). Therefore, I tested 4 aligners with 10 isolate datasets sequenced using Oxford Nanopore Technologies against reference sequences of five single-copy phylogenetic marker genes. Of the 4 aligners tested, minimap2 performed the best when judged by raw and weighted averages of taxonomic distance of alignments to their optimal placements. I subsequently integrated minimap2 and wrote a feature for analyzing long reads in TreeSAPP, and did further testing.</p>

		<p>The workflow performed well with isolate datasets, but poor recall was demonstrated with a mock community, suggesting required improvements in TreeSAPPâ€™s linear model for taxonomic inference, or for higher resolution nucleotide reference packages.</p>
	</div>
	<div class="icon-on-left">
		<simple-svg class="simple-svg" data-icon="whh-fullstar" style="font-size: 5em; margin-right: .3em; float: left;"></simple-svg>
		<b>Genome assembly of previously undescribed symbionts on the pacific sea star <i>Pisaster ochraceus</i></b>
	</div>
	<div>
		<p>My research here focused on the genome assembly of two previously undescribed bacteria, one of which is found on the surface (aboral and oral) of the pacific sea star <i>Pisaster ochraceus</i>, and the other found in the interal cavities (cecum). Preliminary analysis of 16S data have shown that the two organisms, from the genera <i>Spirochaetes</i> and <i>Mollicutes</i> are highly abundant in those respective locations on the sea star, yet their similarity to even the most closest related organisms found in databases are relatively low. Therefore, my role in this project is to assemble these microbes' genomes from metagenomic data from samples of the sea star, and to describe potential metabolic capabilities, functionally annotate the assembled data, and to infer taxonomy. Along the way I'm learning a lot about data preprocessing, assembly, binning, analysis, and writing pipelines. This research was (also) conducted in the <a href="https://www.zoology.ubc.ca/~parfrey/parfrey_lab/">Parfrey lab</a> at UBC, and was funded by an NSERC USRA grant.</p>
	</div>

	<div class="icon-on-right">
		<simple-svg data-icon="whh-plantalt" style="font-size: 5em; margin-right: .3em; float: left;"></simple-svg>
		<b>Functional annotation of microbial communities on the pacific giant kelp <i>Macrocystis pyrifera</i></b>
	</div>
	<div>
		<p>In this project, I assisted with the functional annotation of metagenomic data obtained from the water column adjacent to <i>Macrocystis pyrifera</i>, and the blades of this kelp at differing depths. After data preprocessing, we annotated the cleaned reads using databases of previously annotated genes, of which we focused on carbohydrate active enzymes to explore potential carbon and alginate metabolism, providing the first metagenomic survey of this complex habitat. This was my first exposure to workflow of bioinformatics used in microbial ecology, including quality filtering/trimming of raw data, and exploration/usage of current bioinformatic tools used in the field. This research was conducted in the <a href="https://www.zoology.ubc.ca/~parfrey/parfrey_lab/">Parfrey lab</a> at the University of British Columbia, where I was a work-learn student.</p>
	</div>

	<div class="icon-on-left">
		<simple-svg data-icon="fa-solid:frog" style="font-size: 5em; margin-right: .3em; float: left;"></simple-svg>
		<b>Collation of data on amphibians (frogs) in undisturbed and anthropogenic habitats</b>
	</div>
	<div>
		<p>View the publication <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/ele.13133">here</a>!</p>
		<p>In my first experience with large scale data analysis, I collated data from approximately 80 studies of amphibians, and focused on recording several factors including abundance, location, and taxonomy of frogs. The abundance was recorded in six distinct environments, namely: primary and secondary forest, selectively logged forests, plantations, croplands and pastures in order to observe the impact of different anthropogenic land usage types on the abundance and diversity of frogs living in those respective habitats. My research consisted of searching for, pulling, and entering data from peer-reviewed articles to form a dataset for downstream analysis. Along the way, I developed a couple scripts to help with the data entry, and a script for updating old taxonomic classifications to the latest ones by web scraping the <a href="http://research.amnh.org/vz/herpetology/amphibia/">Amphibian Species of the World database.</a> This research was conducted in the <a href="https://palenlab.wordpress.com/">Palen lab</a> at Simon Fraser University.</p>
	</div>
</div>